### ISSUE TITLE
AWS: Master stop/start fails

### ISSUE THREAD

**Issue by justinsb** [kubernetes/kubernetes#21159]
Although reboot works, we can't stop & start an instance, I think because the ephemeral volume group no longer exists:

From the AWS console:
```
in 27s / 1min 30s)\r\u001b[K[    \u001b[31m*\u001b[1;31m*\u001b[0m] A start job is running for dev-vg\\x...ral.device (1min 28s / 1min 30s)\r\u001b[K[     \u001b[31m*\u001b[0m] A start job is running for dev-vg\\x...ral.device (1min 28s / 1min 30s)\r\u001b[K[    \u001b[31m*\u001b[1;31m*\u001b[0m] A start job is running for dev-vg\\x...ral.device (1min 29s / 1min 30s)\r\u001b[K[   \u001b[31m*\u001b[1;31m*\u001b[0m\u001b[31m*\u001b[0m] A start job is running for dev-vg\\x...ral.device (1min 29s / 1min 30s)\r\u001b[K[\u001b[1;31m TIME \u001b[0m] Timed out waiting for device dev-vg\\x2dephemeral-ephemeral.device.\r\r\n[\u001b[1;33mDEPEND\u001b[0m] Dependency failed for /mnt/ephemeral.\r\r\n[\u001b[1;33mDEPEND\u001b[0m] Dependency failed for Local File Systems.\r\r\n         Starting Wait for all \"auto\" /etc/n... up for network-online.target...\r\r\n         Starting Set console keymap...\r\r\n         Starting Preprocess NFS configuration...\r\r\n[\u001b[32m  OK  \u001b[0m] Stopped Serial Getty on ttyS0.\r\r\n[\u001b[32m  OK  \u001b[0m] Stopped Getty on tty1.\r\r\n[\u001b[32m  OK  \u001b[0m] Closed UUID daemon activation socket.\r\r\n[\u001b[32m  OK  \u001b[0m] Closed ACPID Listen Socket.\r\r\n[\u001b[32m  OK  \u001b[0m] Stopped target Graphical Interface.\r\r\n[\u001b[32m  OK  \u001b[0m] Stopped Accounts Service.\r\r\n[\u001b[32m  OK  \u001b[0m] Stopped target Multi-User System.\r\r\n[\u001b[32m  OK  \u001b[0m] Stopped Terminate Plymouth Boot Screen.\r\r\n[\u001b[32m  OK  \u001b[0m] Stopped Docker Application Container Engine.\r\r\n[\u001b[32m  OK
```

Similar to #7828

**Comment by davidopp** [comment]
Moving to v1.2 P2 to indicate "nice to have" but not release blocker.

**Comment by justinsb** [comment]
This is now fixed in 1.2: restart & stop/start should work reliably.

**Comment by davidopp** [comment]
@justinsb can you indicate which PR fixed it, so we can be sure to cherrypick it? Thanks!

**Comment by justinsb** [comment]
There were a lot of commits before we cut the branch - the whole sync up of kube-up with GCE was partially motivated by this.  Mostly this was just waiting on testing, and blocked by other issues to be fixed before I could test (e.g. Docker 1.9.1 not being installable by Salt etc).   I did find one outstanding issue on Jessie, but there is a dedicated issue & PR for it: #22595   The PR is marked as cherry-pick-candidate and should merge soon.

**Comment by davidopp** [comment]
OK thanks. If you can make sure all the AWS PRs you want in 1.2 are marked cherrypick-candidate, that would be great.

**Comment by justinsb** [comment]
@davidopp  Will do!  There are some where the functionality doesn't really matter either way... should I err on the side of including them for this first batch?  e.g. #21905  - trivial, but I don't know if that means it's better to include it because it would be annoying to have a merge conflict on a future cherry-pick, or whether it's better not to include it because of the effort of cherry-picking it?

**Comment by davidopp** [comment]
Good question. I would err on the side of leaving stuff out, though don't be too shy about including stuff. If something is transitively required by an important cherrypick, we'll figure it out when we try to do the important cherrypick, so don't use that concern as a critiera.
